ImageNet/train/lambda_resnet50_lambda_cloud_single_pytorch.json

```
@@ -0,0 +1,41 @@
+{
+    "version": "v1.0",
+    "author": "Chuan Li",
+    "authorEmail": "c@lambdal.com",
+    "framework": "ncluster / Pytorch 1.0.0",
+    "codeURL": "https://github.com/lambdal/imagenet18/blob/1b43a01af2b8ee5bb9e2405a626d995a750edbb5/training/train_imagenet_nv.py",
+    "model": "Resnet 50",
+    "hardware": "Lambda GPU Cloud - gpu.4x instance",
```

Please add the type of GPU as shown below.

```suggestion
    "hardware": "Lambda GPU Cloud - 4x GTX 1080 Ti",
```

---

We've accepted these changes and updated our fork.